bundles/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/kostalinverter/internal/secondgeneration/SecondGenerationInverterConfig.java

```diff
@@ -25,11 +25,10 @@
 
 @NonNullByDefault
 public class SecondGenerationInverterConfig {
-    public static final long REFRESHINTERVAL_SEC = 60;
-
     public String url = "";
     public String username = "";
     public String password = "";
+    public int refreshInterval;
```

Comment:
Please set it to 60, the default value defined in the XML file.
Note that you should remove the "required" tag for this parameter in the XML file.